components/admin/admin-activity-chart.tsx

```
@@ -72,70 +62,114 @@
             </div>
           </div>
 
           {/* Chart */}
-          <div className="h-48 flex items-end space-x-1">
+          <div className={CHART_CONTAINER_STYLES} role="img" aria-label="Bar chart visualization">
             {data.map((item, index) => (
-              <div key={index} className="flex-1 flex flex-col items-center space-y-1">
+              <div 
+                key={index} 
+                className="flex-1 flex flex-col items-center space-y-1"
+                role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)